### ISSUE TITLE
Indicator computing failed when using the "required" field

### ISSUE THREAD

**Issue by dcoraboeuf** [nemerosa/ontrack#909]
Don't use `toBooleanStrict` for parsing a boolean from a null JSON field:

```
java.lang.IllegalArgumentException: The string doesn't represent a boolean value: null
	at kotlin.text.StringsKt__StringsKt.toBooleanStrict(Strings.kt:1468)
	at net.nemerosa.ontrack.extension.indicators.computing.ConfigurableIndicatorAttributeType$2.invoke(ConfigurableIndicatorAttributeType.kt:29)
	at net.nemerosa.ontrack.extension.indicators.computing.ConfigurableIndicatorAttributeType$2.invoke(ConfigurableIndicatorAttributeType.kt:27)
	at net.nemerosa.ontrack.extension.indicators.computing.ConfigurableIndicatorAttributeType.map(ConfigurableIndicatorAttributeType.kt:42)
	at net.nemerosa.ontrack.extension.indicators.computing.ConfigurableIndicatorType.expandName(ConfigurableIndicatorType.kt:31)
	at net.nemerosa.ontrack.extension.indicators.computing.AbstractConfigurableIndicatorComputer.computeIndicator(AbstractConfigurableIndicatorComputer.kt:32)
	at net.nemerosa.ontrack.extension.indicators.computing.AbstractConfigurableIndicatorComputer.computeIndicators(AbstractConfigurableIndicatorComputer.kt:17)
	at net.nemerosa.ontrack.extension.indicators.computing.IndicatorComputingServiceImpl.compute(IndicatorComputingServiceImpl.kt:22)
```

**Comment by dcoraboeuf** [comment]
Available in 4.0.36

**Comment by dcoraboeuf** [comment]
Same error at:

```
java.lang.IllegalArgumentException: The string doesn't represent a boolean value: null
	at kotlin.text.StringsKt__StringsKt.toBooleanStrict(Strings.kt:1468)
	at net.nemerosa.ontrack.extension.indicators.computing.ConfigurableIndicatorState.getRequiredAttribute(ConfigurableIndicatorState.kt:36)
```

**Comment by dcoraboeuf** [comment]
Available in 4.0.37
